esmvalcore/cmor/check.py

```diff
@@ -402,13 +403,27 @@ def _check_time_coord(self):
             self.report_error(self._does_msg, var_name,
                               'have time reference units')
         else:
+            old_units = coord.units
             coord.convert_units(
                 cf_units.Unit(
-                    'days since 1950-1-1 00:00:00',
+                    'days since 1850-1-1 00:00:00',
                     calendar=coord.units.calendar))
             simplified_cal = self._simplify_calendar(coord.units.calendar)
             coord.units = cf_units.Unit(coord.units.origin, simplified_cal)
 
+            attrs = cube.attributes
+            attr = 'branch_time_in_child'
```

Comment:
`attr` is too generic and used for different (albeit related) purposes below.
I think it would be best to simply repeat the string literal the three times it is needed or choose a more meaningful name for the variable, like `branch_time_child = 'branch_time_in_child'`.

Reply:
Ok, I wanted to avoid having long lines but if it's too confusing I'll rename it.

Reply:
You are absolutely right to avoid long lines! But this should not be at the expense of clarity. Rather prefer introducing line breaks in the argument lists of the function calls.